documents/sidebars.js

```diff
@@ -33,7 +33,7 @@ const sidebars = {
     },
     {
       type: 'category',
-      label: 'ガイドライン',
+      label: 'guidelines',
```

Comment:
`startGuide` の指摘と同じです．

```suggestion
      label: 'Guidelines',
```